apps/cli/src/live-common-setup.ts

```diff
@@ -194,6 +196,8 @@ async function init() {
   });
 }
 
+LiveConfig.setConfig(liveConfigSchema);
```

Comment:
yes, it needs. 
otherwise, a simple "pnpm run:cli" will cause errors.
for example, we can the "sync" command in CLI that can sync a cosmos account. We need to know cosmos config in CLI and web-tools.

"setProviderGetValueMethod" is not necessary. In my current design, if provider is missing, it will take the default value. So it would be good.

Reply:
Just thinking but do we really want to use a default value as a provider :thinking:

Reply:
We don't have config provider in CLI
Default values just means that we use the values in "ledger-live-common/src/families/cosmos/config.ts"

Reply:
Oh ok but then I agree with @haammar-ledger , we need to set the provider in case a node goes down

Reply:
fixed